deck-o-cards.js

```diff
@@ -9,21 +9,35 @@ function deck_o_cards() {
 
   // Make 52 card objects and store them in the "cards" array
   // Hint: use 2 for loops
+  var cardNumb = 1;
 
 
-  // 2. Shuffle the cards
-  // Hint: shuffle function is already defined below
+  for (i=0; i < suits.length; i++){
+      var val = suits[i]
```

Comment:
Setting a variable here isn't necessary because you use it below and it's still accessible from within the 2nd loop. Therefore this is just repeated code.